crates/daphne-service-utils/src/compute_offload/compute_offload.capnp

```diff
@@ -53,10 +53,24 @@ struct PartialDapTaskConfig @0xdcc9bf18fc62d406 {
     vdafVerifyKey       @4  :VdafVerifyKey;
 }
 
+struct PublicExtensionsList @0x8b3c98c0ddd0043e {
```

Comment:
@mendess I just ran `capnp id` and pasted the output here. So this is really just a random ID that needs to be unique?